test/functional/test_site/sub_site/testReuse.md

```diff
@@ -0,0 +1,16 @@
+This is a page from another Markbind site.
+The purpose of this page is to ensure that reuse works as expected.
+All the following images should display correctly.
+
+IMG tags:
+<img src="images/I'm not allowed to use my favorite tool.png">
+<img src="{{baseUrl}}/images/I'm not allowed to use my favorite tool.png">
+<img src="https://dummyimage.com/600x400/000/fff">
+
```

Comment:
Do you think we should also test for using "non-baseUrl" variables as well? E.g.

```
<variable name="somePath">images</variable>
<img src="{{somePath}}/images/I'm not allowed to use my favorite tool.png">
```

What do you think?